build.sh

```diff
@@ -5,6 +5,15 @@ set -o verbose
 
 if [[ $TRAVIS_OS_NAME == "osx" ]]; then
     mysql --version || { brew update && brew install mysql && mysql.server start && mysql --version; }
+    # We need to revert to mysql v 5.7
+    mysql.server stop
+    rm -rf /usr/local/var/mysql
+    brew install mysql@5.7
+    cp -r /usr/local/Cellar/mysql\@5.7/5.7.22 /usr/local/Cellar/mysql/
+    brew unlink mysql
+    brew switch mysql 5.7.22
```

Comment:
This seems fragile and will break when the latest isn't .22 anymore.  Could we instead do:
```
brew install mysql@5.7
brew link mysql@5.7 --force
mysql.server start
```

Reply:
I have updated to force link and travis passes